packages/commonwealth/client/scripts/views/pages/manage_community/chain_metadata_rows.tsx

```diff
@@ -406,6 +450,161 @@ export class ChainMetadataRows extends ClassComponent<ChainMetadataRowsAttrs> {
             m.redraw();
           }}
         />
+        <div class="commonbot-section">
+          <CWText type="h3">Commonbot Settings</CWText>
+          {this.discordBotConnected ? (
+            <>
+              <div class="connected-line">
+                <CWText type="h4">Connection Status</CWText>
+                <div class="connect-group">
+                  <CWIcon iconName="check" iconSize="small" />
+                  <CWText>Connected</CWText>
+                </div>
+              </div>
+              <CWButton
+                label="reconnect"
+                buttonType="mini-black"
+                className="connect-button"
+                onclick={async () => {
+                  try {
+                    const verification_token = uuidv4();
+
+                    await $.post(`${app.serverUrl()}/createDiscordBotConfig`, {
```

Comment:
This is also controller logic.

Reply:
Mind if I just do this in another PR once this is merged?